recipes/findutils/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "findutils" %}
+{% set version = "4.6.0" %}
+{% set sha256 = "ded4c9f73731cd48fec3b6bdaccce896473b6d8e337e9612e16cf1431bb1169d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  script:
+    - ./configure --prefix="{{ PREFIX }}"
+    - make -j${CPU_COUNT}
+    - make install
```

Comment:
Is there a `make check` we can run here?

Also note that as the build script gets bigger and more complex we recommend to use a separate `build.sh` file.

Reply:
I started from the gawk recipe, and it was setup like this. I can move this to a build script.

There is a `make check`. The latest findutils release is three years old and at least one test contains a bug. I was able to find a workaround for this (disabling). A second test fails during the travis-ci build and it is currently not part of the test set.